backend/src/access/index.ts

```diff
@@ -131,7 +131,7 @@ export const generateValidator =
 			if (payloadObj.iss && payloadObj.iss !== certsURL.origin) {
 				throw new Error('JWT issuer is incorrect.')
 			}
-			if (payloadObj.aud && !payloadObj.aud.includes(aud)) {
+			if (payloadObj.aud && !(payloadObj.aud as string).includes(aud)) {
```

Comment:
Instead of casting here, we should cast the construction of the payloadObj on line 124 above with `as JWTPayload`.

(Or even better reuse, somehow, the `getPayload()` function rather than re-parsing it here.)

Reply:
I am not sure how to run and test this code 😓, for that I would just do the `as JWTPayload` here? would that be ok by you?